rfcs/text/0012-orchestrator-field-set.md

```diff
@@ -19,9 +19,68 @@ add a new schema rather than change existing material.
 
 ## Fields
 
-<!--
-Stage 1: Describe at a high level how this change affects fields. Which fieldsets will be impacted? How many fields overall? Are we primarily adding fields, removing fields, or changing existing fields? The goal here is to understand the fundamental technical implications and likely extent of these changes. ~2-5 sentences.
--->
+The proposed change adds six fields, as described below:
+
+```
+---
+- name: orchestrator
+  title: Orchestrator
+  group: 2
+  short: Fields relevant to container orchestrators.
+  description: >
+    Fields that describe the resources which container orchestrators manage or
+    act upon.
+  type: group
+  fields:
+    - name: cluster
+      level: core
+      type: keyword
+      example: production
+      description: >
+        Orchestrator cluster name.
+
+    - name: type
+      level: core
+      type: keyword
+      example: Nomad, Kubernetes
+      description: >
+        Orchestrator cluster type.
+
+    - name: namespace
+      level: core
+      type: keyword
+      example: kube-system
+      description: >
+        Namespace in which the action is taking place.
+
+    - name: resource.name
+      level: core
+      type: keyword
+      example: test-pod-cdcws
+      description: >
+        Name of the resource being acted upon.
+
+    - name: resource.type
+      level: core
+      type: keyword
+      example: service
```

Comment:
Maybe? Do you think we'd see a wide need for users to compare resources from different orchestrator types vs. just finding resources of a type in their particular orchestrator?

Reply:
Yeah, most of the users are probably interested on a particular orchestrator, so probably there is no need to have this set of type-agnostic values.

Another reason to have a known set of values is that this could help to have orchestrator-agnostic dashboards. But such thing is probably not so useful because each orchestrator has its own particularities.

In any case it should be possible to combine resources from different orchestrators taking into account the different values when doing the queries.

So I am ok with continuing without having a fixed list of orchestrator-agnostic values.